src/integrationTest/java/uk/gov/hmcts/reform/unspec/controllers/BaseIntegrationTest.java

```diff
@@ -124,4 +126,9 @@ protected String toJson(Object input) {
             );
         }
     }
+
+    @SuppressWarnings("unchecked")
+    protected Map<String, Object> convertToMap(CaseData caseData) {
+        return (Map<String, Object>) objectMapper.convertValue(caseData, Map.class);
```

Comment:
mapper.convertValue(caseData, new TypeReference<>() {}) would remove the need for unchecked surpression

Reply:
done